browser/js/common/directives/navbar/navbar.js

```diff
@@ -1,47 +1,71 @@
-app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state) {
+app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state, $window) {
 
-    return {
-        restrict: 'E',
-        scope: {},
-        templateUrl: 'js/common/directives/navbar/navbar.html',
+	return {
+		restrict: 'E',
+		scope: {},
+		templateUrl: 'js/common/directives/navbar/navbar.html',
 
-        link: function (scope) {
+		link: function (scope) {
+			var temp = 0;
+			scope.goToSection = function(val, state) {
+				if (val !== 'null') {
```

Comment:
i don't think we need to do all that, especially if we end up with the click-handling route.